static/templates/report_message.hbs

```diff
@@ -0,0 +1,50 @@
+<!-- TODO: add docs and link to them -->
+<!-- future: support tabindex / tabbing to help users reporting multiple messages -->
+<div class="reportmsg-box" class="controls" id="message-history" style="display:none">
+    <div class="radio-input-parent">
+        <label>{{t 'What is the problem with this message?'}}
+            {{> help_link_widget link="/help/stream-permissions" }}
+        </label>
+        <label class="radio">
+            <input type="radio" name="reportmsg-reason" value="spam"/>{{t 'Spam'}}
+        </label>
+        <label class="radio">
+            <input type="radio" name="reportmsg-reason" value="harassment"/>
+            {{t 'Harassment'}}
```

Comment:
I'm surprised our linter doesn't flag this, but I think we usually write this as `{{t 'Harassment' }}` (note the extra space before the `}}`). No function effect, just a style consistency thing.

Reply:
DONE - also changed to double-quotes for consistency.